src/geo/cartodb-layer-group-view-base.js

```diff
@@ -65,7 +65,7 @@ CartoDBLayerGroupViewBase.prototype = {
         tilejson: '2.0.0',
         scheme: 'xyz',
         grids: this.model.getGridURLTemplates(layerIndexInLayerGroup),
-        tiles: this.model.getTileURLTemplates(),
+        tiles: [this.model.getTileURLTemplate()],
```

Comment:
This method is only using other methods from `this.model`, so it could be moved to `src/geo/cartodb-layer-group.js`. I think it'd simplify things a bit... don't you think @donflopez?